src/renderer/components/NodeDocumentationModal.tsx

```diff
@@ -0,0 +1,492 @@
+                                                            );
+                                                        })}
+                                                    </UnorderedList>
+                                                ) : (
+                                                    <Text>This node has no inputs.</Text>
+                                                )}
+                                            </Box>
+                                            <Box position="relative">
+                                                <Heading
+                                                    mb={1}
+                                                    size="sm"
+                                                    userSelect="text"
+                                                >
+                                                    Outputs
+                                                </Heading>
+                                                {selectedSchema.outputs.length > 0 ? (
+                                                    <UnorderedList
+                                                        alignItems="left"
+                                                        ml={0}
+                                                        pl={8}
+                                                        textAlign="left"
+                                                        w="full"
+                                                    >
+                                                        {selectedSchema.outputs.map((output) => {
+                                                            const type =
+                                                                selectedFunctionDefinition?.outputDefaults.get(
+                                                                    output.id
+                                                                );
+
+                                                            return (
+                                                                <ListItem key={output.id}>
+                                                                    <Text
+                                                                        fontWeight="bold"
+                                                                        userSelect="text"
+                                                                    >
+                                                                        {output.label}
+                                                                    </Text>
+                                                                    {output.description && (
+                                                                        <ReactMarkdown
+                                                                            components={ChakraUIRenderer(
+                                                                                customMarkdownTheme
```

Comment:
We are 20 indentation levels deep here. Maybe we could extract the input/output lists into their own components?